events/show.php

```diff
@@ -81,6 +81,19 @@ function GoBack(){
 
   <dt>End Date</dt>
   <dd><?= $event["end"] ?></dd>
+
+<? if(isset($event["location"]["lat"]) && isset($event["location"]["long"]) && isset($GLOBALS["GOOGLE_MAPS_EMBED_API_KEY"])){ ?>
```

Comment:
Taken into account this and your comment on the C# implementation and added `description` in